bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/Page.java

```diff
@@ -227,7 +227,7 @@ default String getTitle() {
     }
 
     /**
-     * If this page is associated with a Template, then this method will return the Template's client libraries categories.
+     * If this page is associated with a Template, then this method will return the Template's client libraries categories for the body tag.
```

Comment:
* client libraries categories -> client library categories
* would be more specific "client library categories to be included before the body end tag"